dts/msm8916/msm8916-samsung-r04.dts

```diff
@@ -19,4 +19,10 @@
 			i2c-address = <0x25>;
 		};
 	};
+	
+	gt58lte-eur {
+                model = "Samsung Galaxy Tab A 8.0 2015 (SM-T355Y)";
+                compatible = "samsung,gt58lte-eur", "qcom,msm8916", "lk2nd,device";
```

Comment:
As far as I can tell, the SM-T355Y is the Australian variant and is `msm8916-sec-gt58lte-aus-r00.dtsi`(i.e. `samsung,gt58lte-aus`) downstream. Do you have the Australian variant maybe?